api/src/db/models/staging/attachment.py

```diff
@@ -0,0 +1,6 @@
+from src.db.models.legacy_mixin import synopsis_mixin
+from src.db.models.staging.staging_base import StagingBase, StagingParamMixin
+
+
+class TsynopsisAttachment(StagingBase, synopsis_mixin.TsynopsisAttachmentMixin, StagingParamMixin):
+    __tablename__ = "tsynopsis_attachment"
```

Comment:
Also needs no underscore
```suggestion
    __tablename__ = "tsynopsisattachment"
```

Reply:
Fixed

Reply:
Still needs to be fixed here